### ISSUE TITLE
[Add request] Twitch.TV

### ISSUE THREAD

**Issue by AnnoyedDev** [blocklistproject/Lists#1136]
URL you wish to be added: https://www.twitch.tv/

Why you believe this should be added: Twitch now officially allow sexual fictionalized content, also what we could consider NSFW or sketchy dances are also allowed based on their last Safety Update : https://safety.twitch.tv/s/article/Updating-our-Approach-to-Sexual-Content-and-Content-Classification-Labels?language=en_US

Add to list: PORN

Other info you think we should know: None.

**Comment by iam-py-test** [comment]
I would like to respectfully disagree; in my opinion, a porn list should contain websites which are primarily for pornography, rather than websites which permit it but serve other purposes. For example, some social media/content sharing sites (X, some mastodon instances, [OnlyFans](https://en.wikipedia.org/wiki/Onlyfans), etc) permit NSFW/pornographic content, but they should not be listed in a pornography list.
Thank you

**Comment by spirillen** [comment]
I agree with @iam-py-test on this one, so luckely we do have such list right here https://github.com/mypdns/matrix/blob/master/source/porn_filters/strict_filters/wildcard.list#L294

If you need any other such domain added, just open an issue at https://kb.mypdns.org/issues/MTX

**Comment by spirillen** [comment]
> Twitch has reverted some part of their policies just now. Lol.

 meaning?? you have been reading the stuff, I have not...

**Comment by spirillen** [comment]
> This rules is not reverted for adult game

Super, leaving the record where it is :smiley:

**Comment by gap579137** [comment]
## Triage Results for #1136
Domain: `[Add request] Twitch.TV`

**DNS:** Failed - [Errno -2] Name or service not known
**HTTP:** 0
**Found in lists:** None

This issue has been automatically triaged by Hermes Agent.

**Comment by blocklistproject** [comment]
This will not be added to `porn`
